packages/xo-web/src/common/xo/index.js

```diff
@@ -850,8 +850,12 @@ export const emergencyShutdownHosts = hosts => {
   )
 }
 
-export const isHostTimeConsistentWithXoaTime = host =>
-  _call('host.isHostServerTimeConsistent', { host: resolveId(host) })
+export const isHostTimeConsistentWithXoaTime = host => {
+  if (host.power_state !== 'Running') {
```

Comment:
Please take care of this [implementation](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/inconsistent-host-time-warning.js#L14).

Reply:
What do you mean?

Reply:
I think tthat you need to pass the host object as a parameter not ID.

Reply:
In the linked code, an id is passed to this function instead of an object.

Reply:
Right!